mk/localdev.mk

```diff
@@ -30,7 +30,7 @@ start-local-psql: check-reg check-venv generate_local_pg_tls_cert
 .PHONY : stop-local
 stop-local:
 	@echo ">stopping local env without deleting containers"
-	$(podmancompose) -f docker-compose.yml -f docker-compose.test.yml stop || $(podmancompose) -f docker-compose.yml -f docker-compose.test.yml stop testrunner phpldapadmin testldap flower redis celery_beat celery_host1 celery_host2 osidb-service osidb-data
+	$(podmancompose) -f docker-compose.yml -f docker-compose.test.yml stop || $(podmancompose) -f docker-compose.yml -f docker-compose.test.yml stop testrunner phpldapadmin testldap flower redis celery_beat osidb_celery_1 osidb_celery_2 osidb-service osidb-data
```

Comment:
`osidb_celery_X` (with X being a number) is the container or host name the automatic replicas will get? It would then be clear that the old names in `setup.mk` do not correspond any more. I am just making sure I understand the context of this change correctly. If so, by `grep` I also get the old name at
https://github.com/RedHatProductSecurity/osidb/blob/master/docker-compose.yml#L51
which has of course no influence on anything be might be later confusing if anyone reads the comment so maybe could you update it accordingly?

Reply:
Turns out I was lazy so I fixed it now.

When the celery containers are running, they are indeed named `osidb_celery_X` as you say.

When podman-compose is invoked, it requires the `celery` name as written in `docker-compose.yml`, not `osidb_celery_X`. That then led to fixing https://github.com/RedHatProductSecurity/osidb/blob/master/docker-compose.yml#L51 properly.

Reply:
Thank you for the explanation!